src/SingleCellAnnotator.imjoy.html

```diff
@@ -955,10 +954,12 @@
       else{
         
         const reasons = [
-          "negative",
-          "out of focus", 
-          "artifact",
-          "missing channels",
+          "Negative",
+          "Unspecific",
+          "Missing channels",
+          "Cell artifacts",
+          "Out of focus", 
+          "Other",
```

Comment:
Remove other and add “Bad segmentation”.

Reply:
Changed